Test test_gjk_libccd-inl_epa.cpp does not compile on Windows due to non-portable M_PI

Similar to https://github.com/flexible-collision-library/fcl/pull/264, probably this went undetected due to ~~https://github.com/flexible-collision-library/fcl/issues/442~~, https://github.com/flexible-collision-library/fcl/issues/389 .

Error log:
~~~
2020-02-23T14:59:10.6836579Z D:\a\drake-lite-cmake\drake-lite-cmake\fcl\test\narrowphase\detail\convexity_based_algorithm\test_gjk_libccd-inl_epa.cpp(260,48): error C2065: 'M_PI': undeclared identifier [D:\a\drake-lite-cmake\drake-lite-cmake\fcl\build\test\narrowphase\detail\convexity_based_algorithm\test_gjk_libccd-inl_epa.vcxproj]
~~~

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,11 @@
 
 ### FCL 0.7.0 (????-??-??)
 
+* Math
+
+  * Replace M_PI instance with constants::pi():
+     [#450](https://github.com/flexible-collision-library/fcl/pull/450)
+
 * Narrowphase
 
   * Various corrections and clarifications of the GJK algorithm used for general

--- a/test/narrowphase/detail/convexity_based_algorithm/test_gjk_libccd-inl_epa.cpp
+++ b/test/narrowphase/detail/convexity_based_algorithm/test_gjk_libccd-inl_epa.cpp
@@ -257,7 +257,8 @@ GTEST_TEST(FCL_GJK_EPA, faceNormalPointingOutwardOriginNearFace1) {
   vertices[1] << 0, 1, face0_origin_distance;
   vertices[2] << -0.5, -0.5, face0_origin_distance;
   vertices[3] << 0, 0, -1;
-  Eigen::AngleAxis<ccd_real_t> rotation(0.05 * M_PI,
+  const double kPi = constants<double>::pi();
+  Eigen::AngleAxis<ccd_real_t> rotation(0.05 * kPi,
                                         Vector3<ccd_real_t>::UnitX());
   for (int i = 0; i < 4; ++i) {
     vertices[i] = rotation * vertices[i];

